windows/ReactTestApp/ReactInstance.cpp

```diff
@@ -0,0 +1,53 @@
+#include "pch.h"
+
+#include "ReactInstance.h"
+
+#include <filesystem>
+
+namespace ReactTestApp
+{
+    void ReactInstance::LoadJSBundleFrom(JSBundleSource source)
+    {
+        const auto &instanceSettings = reactNativeHost_.InstanceSettings();
```

Comment:
Does having `const` here actually work? We're modifying the object here so we shouldn't have it.

Reply:
Non-constant references can't be bound to temporary objects. Agree that we shouldn't use `const` since we modify object. It might become a problem even though it works now. Having `auto &instanceSettings` rises a warning.

Reply:
Interesting. I didn't know it returned a copy. Does it still work with a copy, or do we need to set it back afterwards, i.e. `reactNativeHost_.InstanceSettings(instanceSettings)`?

Reply:
It still works with copy. But now I wonder why it actually does.

Reply:
Huh… I would change it back to just `auto` then.